api/dist/kernel.json

```diff
@@ -1020,7 +1025,7 @@
         "HealthCheckType": "EC2",
         "HealthCheckGracePeriod": "120",
         "MinSize" : "1",
-        "MaxSize" : "100",
+        "MaxSize" : { "Ref" : "MaxClusterSize" },
```

Comment:
Is there any reason to not just set this to a really big number in the first place?